### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Guy-Gustave**
this is the pull request we made on the features branch from the development branch. we still wait for your apreciatin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Todo",
+  "version": "1.0.0",
+  "description": "a JavaScript project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (Dandush03) [inline package.json]: Let's add the following scripts so we can initialize the webpack and build our project

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.9.0",
+    "webpack-cli": "^4.2.0"
+  }
```
Comment (Dandush03) [inline package.json:36]: We are missing here the `webpack-dev-server` dependency. let's include it

File: src/displayProjects.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  const project = Storage.getProjects();
+
+  project.forEach((element, index) => {
+    const project = new ProjectDOMElement(element.title, index);
+
+    projectDisplay.appendChild(project.div);
+  });
+};
+
+export { showProjects };
```
Comment (Dandush03) [inline src/displayProjects.js]: If you're only exporting one object/function in a file, lest add the default flag, this issue repeats in your project, let's fix it. example: `export default showProject`

File: src/displayTodos.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  if (todos.length > 0) {
+    todos.forEach((todo, todoIndex) => {
+      const todoEl = new TodoDOMElement(
+        todo.title,
+        todo.due,
+        todo.priority,
+        todoIndex,
+        todo.done,
+      );
+      todosContainer.appendChild(todoEl.div);
```
Comment (Dandush03) [inline src/displayTodos.js]: Will like to see you're using the destructuring object, I'll give you an example here so you can apply it everywhere
```javascript
todos.forEach((todo, todoIndex) => {
      const {title, due, priority, done} = todo
      const todoEl = new TodoDOMElement(
        title,
        due,
        priority,
        todoIndex,
        done,
      );
      todosContainer.appendChild(todoEl.div);
});
```

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Request Changes
### Great work so far, there is some issue we must fix before moving forward.
- [x] **DOM Buttons:** when trying to modify or delete the project it's very hard to find the actual button since the click isn't been recognize in the hole section of the element. this issue happens when there is more than one ToDo Element
- [x] **Logic:** Please separate the Logic from DOM-Manipulation functions
- [x] **Application:** 
  - [x] The application allows having different projects (or separate lists of to-dos)
  - [x] There is a 'default' project
  - [x] The user can assign a to-do list items to a project
- [x] **Package Json:** [check my comments below](https://github.com/Guy-Gustave/Todo/pull/1#discussion_r533759931)
- [x] **ES6 Syntax:** [check my comments below](https://github.com/Guy-Gustave/Todo/pull/1#discussion_r533764251)

**Comment by visva-dev** [comment]
- Fixed recommendations suggested by tse.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**